README.md

```diff
@@ -76,11 +79,26 @@ You can adjust defaults by passing arguments to `pg-migrate`:
 
 See all by running `pg-migrate --help`.
 
-Most of configuration options can be also specified in `node-config` configuration file.
+Most of configuration options can be also specified in [config](https://www.npmjs.com/package/config) configuration file.
 
 For SSL connection to DB you can set `PGSSLMODE` environment variable to value from [list](https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNECT-SSLMODE) other then `disable`.
 e.g. `PGSSLMODE=require pg-migrate up` ([pg](https://github.com/brianc/node-postgres/blob/master/CHANGELOG.md#v260) will take it into account)
 
+#### JSON Configuration
+
+You can use [config](https://www.npmjs.com/package/config) or your own json file with configuration (`config-file` command line option).
+
+Available options are:
+
+* `migrations-dir`, `migrations-schema`, `migrations-table`, `check-order` - same as above
+
+* either `url` or [`user`], [`password`], `host` (defaults to localhost), `port` (defaults to 5432), `name` - for connection details
+
+* `type-shorthands` - for column type shorthands
+
+  You can specify custom types which will be expanded to column definition (e.g. for `module.exports = { "type-shorthands": { id: { type: 'uuid', primaryKey: true }, createdAt: { type: 'timestamp', notNull: true, default: new require('node-pg-migrate').PgLiteral('current_timestamp') } } }`
```

Comment:
the bracket before "e.g." is not closed here.

Reply:
Can someone provide a real example of how to use this feature? I have been trying in different ways without any success.

Thank you